### ISSUE TITLE
introduce learning rate scheduler

### ISSUE THREAD

**Issue by harbecke** [harbecke/HexHex#30]
Early Bayesian optimization results suggest that optimal parameters are heavily dependent on the training time. However, for most parameters it should be fine to set them to a long training optimum and suffer a small training time increase. This may not be the case for `learning_rate`.
